From: =?utf-8?q?Carl_F=C3=BCrstenberg?= <azatoth@gmail.com>
Date: Tue, 6 Jan 2015 23:12:30 +0100
Subject: Use common license

use common license at /usr/share/common-licenses instead of the shipped
copy
---
 UI/window-license-agreement.cpp | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/UI/window-basic-about.cpp
+++ b/UI/window-basic-about.cpp
@@ -149,15 +149,10 @@
 
 void OBSAbout::ShowLicense()
 {
-	std::string path;
+	std::string path("/usr/share/common-licenses/GPL-2");
 	QString error = "Error! File could not be read.\n\n \
 		Go to: https://github.com/obsproject/obs-studio/blob/master/COPYING";
 
-	if (!GetDataFilePath("license/gplv2.txt", path)) {
-		ui->textBrowser->setPlainText(error);
-		return;
-	}
-
 	BPtr<char> text = os_quick_read_utf8_file(path.c_str());
 
 	if (!text || !*text) {
